static/app/bootstrap/initializeApp.tsx

```diff
@@ -1,6 +1,4 @@
-import 'bootstrap/js/alert';
-import 'bootstrap/js/tab';
-import 'bootstrap/js/dropdown';
```

Comment:
https://github.com/twbs/bootstrap/blob/v3.4.1/js/dropdown.js

Reply:
I looked hard, and couldn't find any actual usages of this?

You can see where we previously used it by running this command `git log -S 'data-toggle="dropdown"' -p`